documentation/contribute/add_a_method.qmd

```diff
@@ -3,14 +3,14 @@ title: Add a method
 order: 40
 ---
 
-This page describes how to add a method to an already existing task. Make sure you have followed the [Requirements](requirements.qmd) and [first setup](first_setup.qmd) pages.
+This page describes how to add a method to an already existing task. Make sure you have followed the [Requirements](requirements.qmd) and [getting started](getting_started.qmd) pages.
```

Comment:
Also mention something like:

    All methods and metrics in OpenProblems consist of Viash components. A Viash component consists of a script and a Viash config which defines the documentation, input/output arguments and dependencies of the script. This page describes how to add a method to an existing task.
    
    :::{.callout-tip}
    Make sure you have followed the [Requirements](requirements.qmd) and [getting started](getting_started.qmd) pages.
    :::

    :::{.callout-note}
    This guide will explain how to add a new method for the Label projection task. Every time you encounter the string `label_projection`, replace it with your task of interest.
    :::